### ISSUE TITLE
Updates to core_templates

### ISSUE THREAD

**Issue by CarlosNZ** [openmsupply/application-manager-server#604]
A place to keep a list of minor (ish) core template config things.

**Comment by CarlosNZ** [comment]
@nmadruga Do you think we should put a few more users in the "core template" snapshot? Only has Admin and nonRegistered to start -- might be useful to have one for each permission name?

**Comment by nmadruga** [comment]
The idea was to have a fresh snapshot to go without any users in a new
configuration. I think we could do something using js files to quickly add
some user/companies to test templates if that's good to have.

**Comment by CarlosNZ** [comment]
I think it's going to be really messy if more than one person works on core_snapshot configs at once. I think we'll need to "check out" the snapshots and then let others know when it's been updated (and probably push straight to develop?)

Also -- whenever someone is going to make changes, they should make a copy before they start in the templates repo, called "core_snapshots_date", so we can easily check old versions if we need to. (We can delete them after a while, just need a way to quickly fix inadvertent regressions)

**Comment by adamdewey** [comment]
Yeah, it's quite interesting eh - it's almost like you need version control within version control haha

**Comment by nmadruga** [comment]
> Also -- whenever someone is going to make changes, they should make a copy before they start in the templates repo, called "core_snapshots_date", so we can easily check old versions if we need to. (We can delete them after a while, just need a way to quickly fix inadvertent regressions)

Yes. That's a great idea. And talking about that - I'm locking **core_templates** now

**Comment by nmadruga** [comment]
Unlocking **core_templates** - changes to UserRego, UserEdit and ResetPassword already commited to develop + previous version snapshot available in `database/old_core_templates/core_templates_2021_10_04`

**Comment by CarlosNZ** [comment]
>     * Core templates should change the application name to something sensible (i.e. not with serial in it) after submission

This doesn't matter at all for "hidden" templates (UserReg, UserEdit, Password), as they shouldn't really be seen in the UI at all.

**Comment by nmadruga** [comment]
I agree, not that relevant. And I only add to those because I was working with these ones first and tyring to see all that was listed to them. So I think it doesn't matter to also have that for the hidden applications in the list (Admin can still see them too).

**Comment by nmadruga** [comment]
Locking `core_templates` again!!

**Comment by nmadruga** [comment]
Question about this one:  `Default Org (FDA, ID 1) missing (and should be linked to Admin user 2)` do we want to add this using the org registration so we can keep the record of the application or just add in the database?

**Comment by nmadruga** [comment]
Unlocking `code_templates`. I have added FDA company, with a logo, but it isn't loading correctly, maybe if you can have a look into that please @CarlosNZ

**Comment by CarlosNZ** [comment]
> Question about this one: `Default Org (FDA, ID 1) missing (and should be linked to Admin user 2)` do we want to add this using the org registration so we can keep the record of the application or just add in the database?

Just add to database. It's the "system org", so shouldn't be connected with any application.

**Comment by CarlosNZ** [comment]
Checking out core_templates....

**Comment by CarlosNZ** [comment]
Returning core_templates. Fixed logo problem and linked permissions to user/org for Admin rather than just org
